code/game/objects/items/weapons/tools.dm

```diff
@@ -640,7 +640,7 @@
 	name = "impact wrench"
 	desc = " The screwdriver's big brother."
 	icon = 'icons/obj/tools.dmi'
-	icon_state = "powerdrill"
+	icon_state = "powerdrillyellow"

```

Comment:
Doesn't the powerdrill randomly assign colours like the screwdriver? Either way, it should work, yah.

Reply:
It would make more sense to fix the powerdrill color choosing logic yeah. I think just doing `drillcolor += "red"` etc would work.